.github/CODEOWNERS

```diff
@@ -0,0 +1,13 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in the repo.
+*       @simonecorsi
```

Comment:
we can add the same list of there: https://github.com/immobiliare/backstage-plugin-ldap-auth/pull/3/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R22 ?